vissl/utils/hydra_config.py

```diff
@@ -285,7 +288,12 @@ def infer_learning_rate(cfg):
         param_schedulers = cfg.OPTIMIZER.param_schedulers.lr
         base_lr = param_schedulers.auto_lr_scaling.base_value
         base_lr_batch_size = param_schedulers.auto_lr_scaling.base_lr_batch_size
+        scaling_type = param_schedulers.auto_lr_scaling.scaling_type
+
```

Comment:
can we add an `assert scaling_type in ['sqrt', 'linear'], "Only linear | sqrt scaling_types are supported"`